### PR TITLE
Creating a new CFTO locator file and moving all the locators in the s…

### PR DESCRIPTION
**Pull request by sumitroy2011**
Created a new JS file to move the CFTO specific locator.
Please review & merge it

### CODE REVIEW

File: spec/cfto/ie/_A_CFTO_IE.test.js [hunk 1]
```diff
@@ -1,15 +1,16 @@
...
     beforeEach(function(done) {
         browser.deleteCookie();
         this.timeout(0);
-        browser.url('http://christmasfood.sit2.marksandspencer.ie');
+        browser.url('http://christmasfood.marksandspencer.ie');
     });
 
 
-    it('Place a CFTO order on the Ireland Site', function() {
+    it.only('Place a CFTO order on the Ireland Site', function() {
```
Comment (SunilSyal) [inline spec/cfto/ie/_A_CFTO_IE.test.js]: Remove only.

**Review by SunilSyal — COMMENTED** [review]
Very nicely done.

File: spec/cfto/ie/_A_CFTO_IE.test.js (hunk 1 above)
Comment (sumitroy2011) [reply spec/cfto/ie/_A_CFTO_IE.test.js]: Incorporated the review comments. Plz review & merge

Comment (sumitroy2011) [reply spec/cfto/ie/_A_CFTO_IE.test.js]: Assigning it to Raminder as Sunil sir is on leave

**Review by SunilSyal — APPROVED** [review]

**Pull request merged**